### ISSUE TITLE
InvalidOperationException

### ISSUE THREAD

**Issue by zirkelc** [microsoft/rate-my-app#41]
InvalidOperationException is thrown by Visual Studio Design.

For more information, see here: http://stackoverflow.com/questions/21469858/wp8-ratemyapp-addon-exception

**Comment by ltomuta** [comment]
This should be fixed already in #29

**Comment by zirkelc** [comment]
Yes, but there is no new build. When will a new version be available?

**Comment by ltomuta** [comment]
We are all pretty busy with other things but this project is not forgotten. Unfortunately I cannot make a promise of when a new release will happen, so my advice would be to clone and build the component on your own.
